pkg/api/v1beta3/conversion_generated.go

```diff
@@ -6289,7 +6446,8 @@ func autoConvert_v1beta3_PodSpec_To_api_PodSpec(in *apiv1beta3.PodSpec, out *api
 	// in.HostIPC has no peer in out
 	// unable to generate simple pointer conversion for v1beta3.PodSecurityContext -> api.PodSecurityContext
 	if in.SecurityContext != nil {
-		if err := s.Convert(&in.SecurityContext, &out.SecurityContext, 0); err != nil {
+		out.SecurityContext = new(api.PodSecurityContext)
+		if err := apiv1beta3.Convert_v1beta3_PodSecurityContext_To_api_PodSecurityContext(in.SecurityContext, out.SecurityContext, s); err != nil {
```

Comment:
Interesting, this was using reflection before which means the custom conversion wasn't even being called.